ports/winpty/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "winpty",
+  "version-string": "0.4.3",
+  "port-version": 2,
```

```suggestion
  "version": "0.4.3",
```
This might be served by version here. https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md

`port-version` is not needed for a new port.